### PR TITLE
Undo/Redo command

### PR DESCRIPTION
**Pull request by NNpanpan**
Basically redo is also there...

### CODE REVIEW

File: src/main/java/seedu/address/model/task/UniqueTaskList.java [hunk 1]
```diff
@@ -87,11 +92,17 @@ public void remove(Task toRemove) {
      */
     public void removeWithModuleCode(final ModuleCode target) {
         List<Task> replacementList = new ArrayList<>();
+        internalList.stream()
+                .filter(task -> !task.getModuleCode().equals(target))
```
Comment (alushingg) [inline src/main/java/seedu/address/model/task/UniqueTaskList.java]: I think this should remove the !

File: src/test/java/seedu/address/testutil/ModelStub.java [hunk 2]
```diff
@@ -303,5 +304,30 @@ public Calendar getCalendar() {
...
+    @Override
+    public DoableAction<?> undo() {
+        return null;
+    }
+
+    @Override
+    public DoableAction<?> redo() {
+        return null;
+    }
+
```
Comment (alushingg) [inline src/test/java/seedu/address/testutil/ModelStub.java]: Maybe follow the top and throw assertion error

Comment (NNpanpan) [reply src/test/java/seedu/address/testutil/ModelStub.java]: Though these should not be a problem, idk why

File: src/main/java/seedu/address/model/task/UniqueTaskList.java (hunk 1 above)
Comment (NNpanpan) [reply src/main/java/seedu/address/model/task/UniqueTaskList.java]: yah

**Pull request merged**